Sources/NetworkProtection/Status/ControlllerErrorMessageObserver/ControllerErrorMesssageObserverThroughDistributedNotifications.swift

```diff
@@ -61,6 +61,8 @@ public class ControllerErrorMesssageObserverThroughDistributedNotifications: Con
         let errorMessage = notification.object as? String
         logErrorChanged(isShowingError: errorMessage != nil)
 
+        Logger.networkProtectionStatusReporter.debug("Received error message")
```

Comment:
The message is not correct.  If `errorMessage == nil` then there's no error message.

Reply:
log removed completely, it was a debugging leftover